LDAR_Sim/install/requirements.txt

```diff
@@ -31,23 +30,20 @@ packaging==23.1
 pandas==2.0.1
 patsy==0.5.3
 Pillow==9.5.0
-pip==23.1.2
+pip==23.3.0
 platformdirs==3.5.0
 plotnine==0.10.1
 pluggy==1.0.0
 ply==3.11
 pooch==1.7.0
 pycodestyle==2.10.0
-pycosat==0.6.4
 pycparser==2.21
 pyOpenSSL==23.1.1
-pyparsing==3.0.9
-PyQt5-sip==12.11.0
-PySocks==1.7.1
-python==3.11.3
+pyparsing
+# PyQt5-sip==12.11.0
+# PySocks==1.7.1
+#python==3.11.0
```

Comment:
Question: Why have these been left in but commented out?

Reply:
Mostly as reference. Will remove, all but the python.